python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -208,7 +215,9 @@ def on_output_generated(self, task_index: int, output: RefBundle):
         if self.obj_store_mem_cur > self.obj_store_mem_peak:
             self.obj_store_mem_peak = self.obj_store_mem_cur
 
-        for block_ref, _ in output.blocks:
+        for block_ref, meta in output.blocks:
+            assert meta.exec_stats and meta.exec_stats.wall_time_s
+            self.udf_time_seconds += meta.exec_stats.wall_time_s
```

Comment:
I'm wondering if this should be called something like "block_generation_time".
Because 1) `wall_time_s` doesn't only include the UDF time; 2) this name is more general and can be used for non-UDF-based ops as well.